ci-operator/step-registry/upi/install/openstack/upi-install-openstack-chain.yaml

```
@@ -0,0 +1,7 @@
+chain:
+  as: upi-install-openstack
+  steps:
+  - ref: upi-install-openstack-inventory
+  - ref: upi-install-openstack-docs
+  documentation: >-
+    This chain follows the documentation for an UPI installation on OpenStack.
```

U has a consonant sound and you should use `a` instead of `an`:
```suggestion
    This chain follows the documentation for a UPI installation on OpenStack.
```